app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,7 @@
   <title>TulOhist</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= analytics_init if (Rails.env.production? and GoogleAnalytics.valid_tracker?) %>
```

Comment:
The logic in this line is included in `app/views/layouts/application.html.erb` and `app/views/layouts/blacklight.html.erb` Could we factor the logic out into a helper? Would make it easier to update in case we need to add other conditions in the future, such as when this is running on devhydra for QA with  RAILS_ENV=production.